### PR TITLE
fix: use css vars for font size (wallet button)

### LINKED ISSUE #2060: font-size overwriting
### Current Behavior

In web3-onboard there are **variables** for **font size**, but for variables are **not used** because fixed values are used for @media

[Line 46](https://github.com/blocknative/web3-onboard/blob/3f391cecd3fa3e77d72598b6080b2bd8bd9da5f1/packages/core/src/views/connect/WalletButton.svelte#L46) set variable for font-size, [Line 86](https://github.com/blocknative/web3-onboard/blob/3f391cecd3fa3e77d72598b6080b2bd8bd9da5f1/packages/core/src/views/connect/WalletButton.svelte#L86) overwrite to hardcode value

image

### Expected Behavior

variables need to be used everywhere

### Steps To Reproduce

1. Open web3-onboard
2. set display width more than 768px
3. look at element-inspector

### What package is effected by this issue?

@web3-onboard/core

### Is this a build or a runtime issue?

N/A

### Package Version

2.7.12

### Node Version

20

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Relevant log output

_No response_

### Anything else?

Because of this problem, I can't customize web3-onboard for my application. In my app 1rem = 10px; and the text in the modal window is too small, I tried to overwrite the data in the variables but found this bug

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

### PR DESCRIPTION
**Pull request by ryanpwaldon**
### Description

Updated `WalletButton` font-size to use css vars instead of a hardcoded value.
- Resolves #2060
- Bumped core version from `2.21.2` → `2.21.3-alpha.1`
- Bumped core version for dependents
  - `packages/demo`
  - `packages/react`
  - `packages/vue`
  - `packages/solid`

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.2",
+    "@web3-onboard/core": "^2.21.3-alpha.1",
```
Comment (Adamj1232) [inline packages/react/package.json:65]: Can you please bump the package version following the alpha release flag up on line 3 of this file?

File: packages/solid/package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.0",
+    "@web3-onboard/core": "^2.21.3-alpha.1",
```
Comment (Adamj1232) [inline packages/solid/package.json:66]: Can you please bump the package version following the alpha release flag up on line 3 of this file?

File: packages/vue/package.json [hunk 3]
```diff
@@ -62,7 +62,7 @@
     "@vueuse/core": "^8.4.2",
     "@vueuse/rxjs": "^8.2.0",
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.2",
+    "@web3-onboard/core": "^2.21.3-alpha.1",
```
Comment (Adamj1232) [inline packages/vue/package.json:65]: Can you please bump the package version following the alpha release flag up on line 3 of this file?

**Comment by ryanpwaldon** [comment]
@Adamj1232 bumped package versions:
- packages/react
- packages/vue
- packages/solid

**Review by Adamj1232 — APPROVED** [review]
Nice work and thank you for your contributions to W3O!

**Pull request merged**